### ISSUE TITLE
Question about training set

### ISSUE THREAD

**Issue by mukulkhanna** [StanfordVL/iGibsonChallenge2021#18]
Hi team,

I am working with the iGibson scene dataset for a project and am interested in using the train/test/val splits being used for the iGibson challenge for our task.

I see that for the challenge, 8 scenes have been chosen for the train split with their [visualization](https://github.com/StanfordVL/iGibsonChallenge2021/raw/master/images/cvpr21_dataset.gif) provided in the README. I also read in the README ([here](https://github.com/StanfordVL/iGibsonChallenge2021#using-docker)) that the `Rs_int` scene is one of the training scenes. 

However, I am not sure if that is correct. I have worked with the scenes enough to be able to identify which scenes these are (by name). I doubt that `Rs_int` scene belongs to the train set. Can someone please confirm this? Thanks :)

**Comment by ChengshuLi** [comment]
Hi @mukulkhanna, I confirmed that `Rs_int` is one of the training scenes. The visualization provided in the README is not accurate. Thank you!

**Comment by mukulkhanna** [comment]
Thanks @ChengshuLi! As I mentioned, I am working with the iGibson scene dataset for a project and am interested in using the train/test/val splits for our task.

Could you share which scene `Rs_int` is replacing in the visualization?

**Comment by ChengshuLi** [comment]
@mukulkhanna I would prefer not to share the train/test/val split in public in case people train on the test set. Could you provide your email to me? I am happy to share it with you in private.

**Comment by mukulkhanna** [comment]
@ChengshuLi I just emailed you for the same. Thanks!
